### DISCUSSION TITLE
Can an example of the JSON syntax for using 'java' or 'js' configuration options be provided?

### DISCUSSION THREAD

**Discussion by JoeJennings** [renovatebot/renovate#20054] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

2.6.0

### If you're self-hosting Renovate, select which platform you are using.

GitHub Enterprise Server

### If you're self-hosting Renovate, tell us what version of the platform you run.

3.6.3

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Would like to leverage the 'java' configuration option: https://docs.renovatebot.com/configuration-options/#java to set -Xmx and -Xms

### What you tried so far.

I've tried a few different formats and the configuration validation doesn't seem to like them.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
That's not what the option is for

**Reply by JoeJennings** [reply]
I apologize for my ignorance, is there some documentation or examples that illustrate/explain the usage/purpose? Or is it as simple as it's for other Renovate configuration options to be used across any given java project/repository using a java package manager?

**Reply by rarkins** [reply]
I recommend you stick to packageRules instead. Eg matchManagers, matchDatasources, etc

**Reply by JoeJennings** [reply]
Got it, Thanks!
